internal/commands/push/command.go

```diff
@@ -198,7 +198,35 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		if err := clients.Realm.ImportDependencies(appRemote.GroupID, appRemote.AppID, uploadPathDependencies); err != nil {
 			return err
 		}
-		ui.Print(terminal.NewTextLog("Uploaded dependencies archive"))
+
+		ui.Print(terminal.NewTextLog("Uploaded dependencies"))
+		s := spinner.New(terminal.SpinnerCircles, 250*time.Millisecond)
+		s.Suffix = " Installing dependencies: starting..."
+
+		s.Start()
+		defer s.Stop()
+		waitForInstallation := func() error {
```

Comment:
[naming nit] lets rename this to `installDependencies`/`importDependencies`/something along those lines

also, I think I mentioned this earlier but we should include the `clients.Realm.ImportDependencies(...)` call inside this function as well.  this way, we have the spinner present (and only present) for the duration of these dependencies actions)

Reply:
I see; I misinterpreted that comment and changed the spinner to outside the scope the nested function instead.